src/atom-environment.coffee

```diff
@@ -4,6 +4,7 @@ path = require 'path'
 
 _ = require 'underscore-plus'
 {deprecate} = require 'grim'
+environment = require('./environment')
```

Comment:
This is a weird name to me... what do you think of `environmentHelpers` or something?